### PR TITLE
Updated Bootstrap 4.0 & a little login feature

### PR DESCRIPTION
**Pull request by JStuve**
### Bootstrap Upgrade
* Switch our project to Bootstrap 4.0
* Changed the preexisting elements

### Auth0 Integration
* Added login option to navbar
* Not sure if the navbar is permanent so this might be temporary
* Login is still not required to use the app
* **THIS IS NOT FINAL STATE**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "@angular/platform-browser": "^5.2.0",
     "@angular/platform-browser-dynamic": "^5.2.0",
     "@angular/router": "^5.2.0",
+    "angular2-jwt": "^0.2.3",
```
Comment (cvasqu09) [inline package.json]: Unable to build the project. Getting the following error. Not sure if we need to include install other dependencies?
image

File: src/app/auth.service.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
+import { Injectable } from '@angular/core';
+import 'rxjs/add/operator/filter';
+import * as auth0 from 'auth0-js';
```
Comment (cvasqu09) [inline src/app/auth.service.ts]: Do we need to install auth0-js? This is the file that is causing issues with the image posted above.

File: src/app/auth.service.ts [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+@Injectable()
+export class AuthService {
+
+  auth0 = new auth0.WebAuth({
+    clientID: 'YdXJArBHl5cZiJP9Fjp4aCv2wd4LsYBg',
```
Comment (cvasqu09) [inline src/app/auth.service.ts]: Do we want this information stored as environment variables? Can always change this later. No need to worry about this atm.

Comment (JStuve) [reply src/app/auth.service.ts]: It's okay to have the clientID public, outside users will not be able to retrieve/configure anything in regards to that client ID without actually logging onto the Auth0 profile.

File: src/app/auth.service.ts (hunk 2 above)
Comment (JStuve) [reply src/app/auth.service.ts]: Yes. `npm install auth0-js` should work

Comment (cvasqu09) [reply src/app/auth.service.ts]: I just added it as a dependency so that building it when we deploy goes through.

**Comment by cvasqu09** [comment]
Everything looks good to go.

**Pull request merged**